src/pyedb/dotnet/edb_core/components.py

```diff
@@ -981,16 +1016,23 @@ def create_port_on_component(
             for p in ref_pins:
                 if not p.IsLayoutPin():
                     p.SetIsLayoutPin(True)
-            if len(ref_pins) == 0:
-                self._logger.info("No reference pin found on component {}.".format(component.GetName()))
+            if not ref_pins:
+                self._logger.warning(
+                    "No reference pins found on component, extending pin search at the component "
+                    "vicinity Pingroups are not supported, the closest pin will be selected."
```

Comment:
```suggestion
                    "No reference pins found on component, the closest pin will be selected."
```
The message is not clear to me, what about this one ? Less information but at least I'm not lost when reading it.